monshan/rancid-tomatillos
# Build poster view

# PR Rancid Tomatillos
**Connor Anderson-Larson & Marika Shanahan**

### Fix or Feature?


### Are there any known bugs?


### Summary of the changes
Add single poster view on movie click. User can now select a movie to see more information and return to main page.
In doing this we found it pertinent to add API data while making the poster update dynamic.


### What ticket(s) do the changes resolve?

User can click a movie to see details
User can return to the main menu

### Notes for the Reviewer

Site should be fully functional with what is available currently on site.
Responsiveness has yet to be tested for poster view.

### How to Test Changes?

Run npm start on app after cloning,
Observe UI:
* Clicking a movie should bring you to a new page
* Clicking return button should bring you to the main theatre

### Next Iterations
* Add responsiveness/media query to Single Poster View
* Refactor with Router
* Improve accessibility with HTML syntax
* Move network requests to separate file to improve readability
